apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -37,6 +37,24 @@ $(document).ready(() => {
   init();
 
   function init() {
+    // TO-DELETE ONCE OPTIMIZELY-EXPERIMENT IS COMPLETE (start)
+    let inVariant = false;
+    const queryString = window.location.search.substring(1).split('&');
+    for (let i = 0; i < queryString.length; i++) {
```

Comment:
You could also use `.find` here rather than looping through the query string array.
ex. `const inVariant = queryString.find(param => param === 'inSignUpUserTypeVariant=true');`

Reply:
Oh good point! I may actually use .includes() instead since that will give me a boolean

Reply:
Even better!